test/choosetests.jl

```diff
@@ -80,7 +80,7 @@ function choosetests(choices = [])
         prepend!(tests, linalgtests)
     end
 
-    net_required_for = ["socket", "parallel"]
+    net_required_for = ["socket", "parallel", "libgit2"]
```

Comment:
this seems fine, I rarely see net_on not true. WSL is one of the few places that's the case